### DISCUSSION TITLE
Is it possible to obtain structural system type of a building with BRAILS?

### DISCUSSION THREAD

**Discussion by serkanhasanoglu** [NHERI-SimCenter/SimCenterCommon#224] [Brails++] [answered]
Hello, I just wonder that is the algorithm of BRAILS somehow capable of getting information about the structural system type of a building (RC Wall, RC-MRF, RC-Dual, etc.)

**Comment by bacetiner** [comment, marked as answer]
Thank you for your question, and I apologize for the delayed response. Currently, BRAILS cannot differentiate between various reinforced concrete structural systems. It can only classify construction types at a higher level into the categories: 1) reinforced concrete, 2) steel, 3) masonry, 4) wood, and 5) manufactured buildings.

I hope this helps!
